pal/audio/web/player-web.ts

```diff
@@ -95,7 +95,10 @@ export class AudioContextAgent {
                 resolve();
                 return;
             }
-            context.resume().catch((e) => { debug.warn('runContext error', e); });
+            // only state is suspended can involve resume function
+            if (context.state === 'suspended') {
+                context.resume().catch((e) => { debug.warn('runContext error', e); });
+            }
             if (context.state === 'running') {
```

Comment:
The 'if' condition needs to be changed to 'else if'.

Reply:
ok